### PR TITLE
test(e2e): changes done in search,ssi,usergroup spa to support new features (ui changes)

### PR DESCRIPTION
**Pull request by mudit94**
…atures

# Closes/Fixes/Resolves
changes done in search,ssi,usergroup spa to support new features (ui changes)

# Explain the feature/fix

changes done in search,ssi,usergroup spa to support new features (ui changes)

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demoed and the design review approved?

### CODE REVIEW

File: e2e/integration/usergrouptests.js [hunk 1]
```diff
@@ -51,13 +51,13 @@ context( 'Test for UserGroup', () => {
...
     it( 'Test for trying to add new group with existing name', () => {
-        cy.get( 'a[href="/user-groups"]' ).click()
+        cy.get( '.pf-c-breadcrumb__item  a[href="/user-groups"]' ).click()
         cy.get( '#username', { timeout: 5000 } ).type( Cypress.env( 'USERNAME' ) );
         cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
         cy.get( '#submit' ).click();
         cy.get( 'a[href="/user-groups/group/new"]' ).click();
         cy.get( '#groupName' ).type( 'test1' );
-        cy.get( '[aria-label="Search input"]' ).type( 'one - portal - devel' );
+        cy.get( '[aria-label="Search input"]' ).type( 'one - portal - devele' );
```
Comment (riginoommen) [inline e2e/integration/usergrouptests.js:60]:
```suggestion
        cy.get( '[aria-label="Search input"]' ).type( 'one - portal - devel' );
```

**Review by deshmukhmayur — APPROVED** [review]
Other than Rigin's comment, lgtm

File: e2e/integration/usergrouptests.js (hunk 1 above)
Comment (mudit94) [reply e2e/integration/usergrouptests.js]: fixed @riginoommen https://github.com/1-Platform/one-platform/pull/1455/commits/398510661fece6497d46559fb4546913750f45ad

**Review by riginoommen — APPROVED** [review]

**Comment by mudit94** [comment]
> Other than Rigin's comment, lgtm

lgtm is passed. rigin's comment has been resolved . :)

**Review by deshmukhmayur — APPROVED** [review]

**Pull request merged**